src/app/screens/sign-up/sign-up.component.html

```diff
@@ -0,0 +1,36 @@
+<app-form>
+
+  <form (ngSubmit)="signUp()">
+    <div class="content-input">
+      <label for="name">Nombre</label>
+      <input type="text" [(ngModel)]="signUpForm.name" [ngModelOptions]="{ standalone: true}">
+    </div>
+
+    <div class="content-input">
+      <label for="lastname">Apellido</label>
+      <input type="text" [(ngModel)]="signUpForm.lastname" [ngModelOptions]="{ standalone: true}">
+    </div>
+
+    <div class="content-input">
+      <label for="email">Email</label>
+      <input type="email" [(ngModel)]="signUpForm.email" [ngModelOptions]="{ standalone: true}">
+    </div>
+
+    <div class="content-input">
+      <label for="password">Password</label>
+      <input type="password" [(ngModel)]="signUpForm.password" [ngModelOptions]="{ standalone: true}">
+    </div>
+
+    <div class="content-input">
+      <label for="confirmPassword">Confirmación de Password</label>
+      <input type="password" [(ngModel)]="signUpForm.confirmPassword" [ngModelOptions]="{ standalone: true}">
+    </div>
+
+    <div class="section-buttons">
+      <button class="btn-fill" type="submit">Sign Up</button>
+      <div class="line-button"></div>
```

Comment:
Remember to use as less html code as possible for clarity and reusability. Try to use the decorative part via css only and just the structure and functionality parts in the html, for this case you can use pseudo elements.